etcdctl/ctlv2/command/util.go

```diff
@@ -78,7 +79,7 @@ func getPeersFlagValue(c *cli.Context) []string {
 
 	// If we still don't have peers, use a default
 	if peerstr == "" {
-		peerstr = "http://127.0.0.1:2379,http://127.0.0.1:4001"
```

Comment:
@heyitsanthony yes it should work. But i had delibrately thought of modifying it specifically for windows because of etcd current logging mechanism which prints all urls in form of ip:port. And i felt it would be better to keep the logs uniform with the display options as far as possible.
Let me know if you feel its better to keep it localhost for all will do the changes accordingly.